app/api/listings/route.ts

```
@@ -0,0 +1,297 @@
+import { NextResponse } from "next/server";
+import { getSupabaseRouteHandler } from "@/utils/supabase/server";
+import { title } from "node:process";
+
+export async function GET(request: Request) {
+  const supabase = await getSupabaseRouteHandler();
+  const { searchParams } = new URL(request.url);
+
+  const id = searchParams.get("id");
+  const page = parseInt(searchParams.get("page") || "1", 10);
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)